extensions/historical-imagery/schema.json

```diff
@@ -164,30 +164,30 @@
             "platform": {
               "type": "array",
               "items": {
-                "type": "string",
-                "minItems": 1
-              }
+                "type": "string"
+              },
+              "minItems": 1
             },
             "mission": {
               "type": "array",
               "items": {
-                "type": "string",
-                "minItems": 1
-              }
+                "type": "string"
+              },
+              "minItems": 1
             },
             "instruments": {
               "type": "array",
               "items": {
-                "type": "string",
-                "minItems": 1
-              }
+                "type": "string"
+              },
+              "minItems": 1
             },
             "proj:epsg": {
               "type": "array",
               "items": {
-                "type": "integer",
```

Comment:
I just realised that `null` is a valid value for proj:epsg. so minItems might be an issue. @MDavidson17 @l0b0

Reply:
Fixed; thanks for noticing!

Reply:
good spot! When we allow for null do we want the summaries to look like: `proj:epsg:[]` or `proj:epsg: [null]`?

If the second case we could create a new issue to implement something like @l0b0 's suggestion [here](https://github.com/linz/stac/pull/176#discussion_r753907952) (I'm not sure this link quite works see copy of the comment below)

> Does this actually allow `null` though? I think you might need something which is explicitly optional, like
> 
> ```
> {
>   "oneOf": [
>     {
>       "type": "string",
>       "format": "date-time",
>       "pattern": "(\\+00:00|Z)$"
>     },
>     {
>       "type": "null"
>     }
>   ]
> }
> ```

Reply:
I don't think we should have an explicit `null` in the list. IMO we should make `proj:epsg` *optional* instead of allowing `null`.